lib/puppet/resource/status.rb

```diff
@@ -101,7 +101,7 @@ def dependency_failed?
       YAML_ATTRIBUTES = %w{@resource @file @line @evaluation_time @change_count
                            @out_of_sync_count @tags @time @events @out_of_sync
                            @changed @resource_type @title @skipped @failed
-                           @containment_path}.
+                           @containment_path @corrective_change}.
```

Comment:
This change is only there to ensure that the `#to_yaml_properties` and `#to_data_hash` uses the same property set. The YAML report is always produced using `#to_data_hash` (tested with Ruby 1.9.3 and 2.3.1) so it is unaffected by this change.